Userland/Applications/Browser/BrowserWindow.cpp

```diff
@@ -546,6 +546,10 @@ void BrowserWindow::create_new_tab(URL url, bool activate)
         return m_cookie_jar.get_all_cookies();
     };
 
+    new_tab.on_get_local_storage_objects = [this]() {
+        return active_tab().m_web_content_view->get_local_storage_objects();
```

Comment:
I feel like "objects" is the wrong term for this - they're no objects, they're key/value pairs, or simply "entries" (https://infra.spec.whatwg.org/#map-entry):

> To reorder a [Storage](https://html.spec.whatwg.org/multipage/webstorage.html#storage-2) object storage, reorder storage's [map](https://html.spec.whatwg.org/multipage/webstorage.html#concept-storage-map)'s [entries](https://infra.spec.whatwg.org/#map-entry) in an [implementation-defined](https://infra.spec.whatwg.org/#implementation-defined) manner.

and

> Each key/value pair is called an entry.

Reply:
Thanks for looking that up! I didn't even think to look at the spec for this.